chart/templates/horizontal-pod-autoscaler.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if and .Values.hpa.enabled (or (.Values.hpa.triggers.cpu.enabled) (.Values.hpa.triggers.memory.enabled)) }}
```

Comment:
What was the thinking behind including `hpa.triggers.cpu.enabled` and `hpa.triggers.memory.enabled` here? (over just having single flag `hpa.enabled`)

Reply:
> What was the thinking behind including `hpa.triggers.cpu.enabled` and `hpa.triggers.memory.enabled` here? (over just having single flag `hpa.enabled`)

nvm, I think I see it. This is to make sure in addition to `hpa.enabled` if one of the metric resources are enabled as well, right?

On that note, rather than introducing a separate `enabled` flag for each what if we just check if a value is defined for either `.Values.hpa.triggers.cpu.threshold` or `.Values.hpa.triggers.memory.threshold`   ?

Reply:
> nvm, I think I see it. This is to make sure in addition to hpa.enabled if one of the metric resources are enabled as well, right?
Correct.

> On that note, rather than introducing a separate enabled flag for each what if we just check if a value is defined for either .Values.hpa.triggers.cpu.threshold or .Values.hpa.triggers.memory.threshold ? 
This will work. However, if I go this route I will not be able to provide default values for cpu threshold since we want hpa off by default.

Reply:
>This will work for rendering purposes. However, if I go this route I won't be able to provide default values since the AC says to disable the HPA controller by fault. Consequently, I would not be able to fulfill part of the AC:

Fair enough :+1: Let's keep it as is for now. (Was only looking way to reduce config variables, but I don't think this will be a huge concern for anyone. we'll just have to document these clearly)

Reply:
Addressed: https://github.com/sonatype/nexus-iq-server-ha/pull/15/commits/0a339f7c0fc27492603b4234ffa26e69054358b9